content/references/moderator-features.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Let's put this next to `moderation.md` and name it `moderation-tools.md`.

And it can be linked from line 39 instead:

> They have earned the trust of the admins from years of contributions and are given access to [moderation tools] and extra privileges.

---

`/reference/` is for when it doesn't fit under anywhere, and it's better to avoid it.

Reply:
Done.

I've updated also navigation: next/prev links, and sidebar. Let me know if it's OK now